invenio_records/signals.py

```diff
@@ -69,3 +69,16 @@ def listener(sender, *args, **kwargs):
 
 after_record_update = _signals.signal('after-record-update')
 """Signal sent after a record is updated."""
+
+before_record_index = _signals.signal('before-record-index')
```

Comment:
I think you can rebase and you will get this? It is now part of the `master` branch